examples/python/open3d_tutorial.py

```diff
@@ -261,7 +261,7 @@ def get_eagle_pcd():
     path = _relative_path("../test_data/eagle.ply")
     if not os.path.exists(path):
         print("downloading eagle pcl")
-        url = "http://www.cs.jhu.edu/~misha/Code/PoissonRecon/eagle.points.ply"
```

Comment:
I would keep their original sources, as it is already a direct link. Also for copyright reasons.